scripts/run-live-build.sh

```diff
@@ -15,6 +15,12 @@
 # limitations under the License.
 #
 
+#
+# The appliance-build upstream branch. This should be updated by the release
+# lead on branching.
+#
+BRANCH="master"
```

Comment:
We also will need to update the line in `build-ancillary-repository.sh`.. I'd be great to consolidate both uses into a single place, if possible.. if not, please also update any release docs to also reference that other script (or ask Pavel to do that, since he originally added that other line), so that it gets updated in addition to this line.